### DISCUSSION TITLE
GitHub Meldung "main branch isn't protected"

### DISCUSSION THREAD

**Discussion by yvonneadam** [neuefische/questions#44] [Web] [answered]
Hat noch jemand diese Meldung, immer wenn man mehr als nur die main branch offen hat? 

<img width="2000" alt="Bildschirmfoto 2022-08-02 um 19 11 46" src="https://user-images.githubusercontent.com/106114808/182434618-275bdb33-c3e5-4c3e-abe8-44e1a88083c6.png">

Iszt das wichtig zu beachten und was müsste daraufhin unter PROTECT THIS BRANCH eingestellt werden?

**Comment by pixelass** [comment, marked as answer]
Du kannst deinen Branch sichern (also gewisse Regeln erstellen die erfüllt werden müssen um auf `main` zu pushen oder mergen. Das ist vor allen für Open Source oder Kollaboration wichtig. 
Diese Meldung könnt ihr gerne ignorieren.

Wenn du das einrichten willst, schau gerne in den Settings deines Repos: 

<img width="1234" alt="Screenshot_02_08_22__19_28" src="https://user-images.githubusercontent.com/1148334/182437290-a410af17-0da5-4eb0-9889-b115bf85482e.png">
